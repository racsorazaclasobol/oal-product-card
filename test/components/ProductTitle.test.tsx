import React from 'react';
import rendered from 'react-test-renderer';
import ProductCard, { ProductTitle } from '../../src/components';
import { product1 } from '../data/products'

describe('ProductTitle', () => {
    test('Debe de mostrar el componene correctamene con el titulo personalizado', () => {

		const wrapper = rendered.create(<ProductTitle title="Test Product" />);

		expect(wrapper.toJSON()).toMatchSnapshot();

    });

	test('Debe de mostrar el componene correctamene con el titulo del producto', () => {

		const wrapper = rendered.create(
			<ProductCard product={ product1 }>
				{
					() => (
						<ProductTitle />
					)
				}
			</ProductCard>
		);

		expect(wrapper.toJSON()).toMatchSnapshot();
		
    });
});
