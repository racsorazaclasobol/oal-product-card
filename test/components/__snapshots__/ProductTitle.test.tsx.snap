// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductTitle Debe de mostrar el componene correctamene con el titulo del producto 1`] = `
<div
  className="productCard undefined"
>
  <span
    className="productDescription undefined"
  >
     
    Coffee Mug - Sin Imagen
     
  </span>
</div>
`;

exports[`ProductTitle Debe de mostrar el componene correctamene con el titulo personalizado 1`] = `
<span
  className="productDescription undefined"
>
   
  Test Product
   
</span>
`;
